src/main/resources/data/invoices/12345_paid.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "6b8bc989-834d-4a14-945b-4c5442ae09af",
+  "adjustments": [
+    {
+      "description": "Sales Tax",
+      "type": "Percentage",
+      "value": 5.00,
+      "prorate":"Not prorated",
```

Comment:
Invoice level adjustments that are not prorated must have a fund distribution from which the amount will be deducted
Ref: https://github.com/folio-org/acq-models/pull/136#discussion_r287111825

Reply:
Done